test/test_queries.py

```
@@ -21,13 +28,63 @@ def test_strings_load_embedding_via_json_filename() -> None:
+    video_embeddings = pd.read_feather(video_embeddings_path)
+
+    df = load_embedding_dataframe(embedding_collection, video_embeddings)
+
+    assert isinstance(df, pd.DataFrame)
+    assert df.shape == (2, 512)
+    assert math.isclose(np.linalg.norm(df.T[0]), 1.0, rel_tol=0.0001, abs_tol=0.0001)
+    assert math.isclose(np.linalg.norm(df.T[1]), 1.0, rel_tol=0.0001, abs_tol=0.0001)
+
+
+def test_frame_load_embedding_via_json_filename_multi_embeddings(
+    video_path: Path, video_embeddings_path: Path
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)